### PR TITLE
Add changelog entry for #16070

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -43,6 +43,7 @@
 - Change: [#16710] Changed default view of Guest List to 'Thoughts' and selected tab will default to 'Summarised' (when opened from the menu).
 - Change: [#16859] Guests with umbrellas no longer always avoid going into a Maze.
 - Change: [#16912] Tired or nauseated guests will no longer jump in a Maze.
+- Fix: [#6767] Black screen after alt-tab.
```
Comment (Broxzier) [inline distribution/changelog.txt:46]:
```suggestion
- Fix: [#6767] Black screen after Alt-Tabbing back into the game.
```

**Comment by MaxMallon** [comment]
Shouldn't at least my two bugfixes be included in 0.4.0?

**Comment by janisozaur** [comment]
@MaxMallon I don't understand what your comment has to do with this pr? If you feel there's something that's missing from changelog, you can submit pr easily, even from the web UI.

**Comment by MaxMallon** [comment]
> 

I don't know how I got here. I saw a page with a list of PRs/issues to be added in 0.4.0, and saw my fixes weren't among them, and decided to comment, yet that somehow ended up here... 
This page: https://github.com/OpenRCT2/OpenRCT2/milestone/33

**Pull request merged**